Makefile.ci

```diff
@@ -37,7 +37,6 @@ CI_TARGETS= \
     ci-fiat_crypto_legacy \
     ci-fiat_crypto_ocaml \
     ci-fiat_parsers \
-    ci-flocq3 \
```

Comment:
also delete the script and basic-overlay section

Reply:
Done